docs/layouts/_default/baseof.html

```diff
@@ -23,5 +23,143 @@
     {{ block "sidebar-footer" . }}{{ end }}
     {{ partial "footer/footer.html" . }}
     {{ partial "footer/script-footer.html" . }}
+
+    <!-- Google Tag Manager -->
+    <script type="text/plain" data-cookiecategory="analytics">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','GTM-TR443T9');</script>
+    <!-- End Google Tag Manager -->
+
+    <!-- Cookie Consent -->
+    <script defer src="https://cdn.jsdelivr.net/gh/orestbida/cookieconsent@v2.8.0/dist/cookieconsent.js"></script>
```

Comment:
Can we install this lib as a dependency/vendor and import from source instead of using a cdn? cdns are great for PoCs but not so reliable for prod.

Reply:
Done.